### PR TITLE
add bios of academic track core team

### PR DESCRIPTION
**Pull request by tyrasd**
This would add some background info about the members of the academic track team to the website.

preview: https://giscience.github.io/stateofthemap-2019/program/#academic-track

Questions:
* do we even want this on the website?
* should it be on a different page? the program page will later probably contain more stuff like the program schedule, so the page could get crowded at some point. on the other hand we have the left hand side menu now, so this should be less of an issue…
* should we also add email addresses?
* should we also add current affiliations?
* should we tweak the bio texts? e.g. sometimes the bio starts with "first name", sometimes it starts more formal: Dr. "full name", some of them could maybe be a bit shorter, etc.

//cc @MarcoMinghini @yair-grinb @ChristineKarch

### CODE REVIEW

**Comment by Nakaner** [comment]
> * do we even want this on the website?

I want it! Being transparent about the programme committee (in this case the academic one) is great.

I would like to ask the non-academic programme committee to do the same (although they don't have to have photos because I don't want to force any member to agree).

> * should it be on a different page? the program page will later probably contain more stuff like the program schedule, so the page could get crowded at some point. on the other hand we have the left hand side menu now, so this should be less of an issue…

The programme committees should be on a page together with the call for papers. In 2018, there was a "Call for Papers" link in the main menu pointing to the blog entry at blog.openstreetmap.org. On 25 January 2019 (Git commit 23923cf2701be159780c1fb8168c0035ff33f67d) the page looked like this:

bildschirmfoto_2019-02-06_21-59-12

The Program page had following content:

```md
State of the Map 2018 is taking place from **July 28th to 30th 2018** in Milan, Italy. We are looking forward to designing a program with you - the OpenStreetMap community. You only have until 14th February to <a href="https://blog.openstreetmap.org/2018/01/11/session-proposals-sotm-2018/">submit your session proposals</a> and until 4th March to <a href="https://2018.stateofthemap.org/academictrack/">submit your academic abstract</a> so don't delay!

In the meantime, here is the outline of the program:

- July 28th: first main conference day with presentations and discussions
- July 29th: second main conference day with presentations and discussions
- July 30th: activity day with code, documentation sprints, mapping and workshops

We plan to have two tracks of talks throughout the first two days, with the option to expand to three tracks for a half day of <a href="https://2018.stateofthemap.org/academictrack/">academic talks</a>. Breakout rooms will be available throughout.

*Your State of the Map organizing team.*
```

> * should we also add email addresses?

I don't think that this is necessary. A email address to the whole committee is sufficient. If someone wants to contact someone privately, they can search for the person and will find an email address. (They all applied publicly ;-) )

> * should we also add current affiliations?

Yes. You could include them in the bio as it is already the case with some of them already.

> * should we tweak the bio texts? e.g. sometimes the bio starts with , sometimes it starts more formal: Dr. , some of them could maybe be a bit shorter, etc.

I am fine with them as they are. OSM is diverse, isn't it? ;-)

File: program/index.md [hunk 1]
```diff
@@ -23,6 +23,14 @@ We are looking forward to further designing our program with you and welcome you
 
 We expect to share our call for academic and general session proposals in the beginning of <span class="numeral">2019</span> - <a href="{{site.baseurl}}/#newsletter">sign up to our newsletter</a> for event updates.
 
+### Academic Committee
+
+* <img style="height:160px; float:right; clear:right; margin-left: 1em; margin-bottom: 0.7em; margin-top: 1.7em; filter:grayscale(1);" src="../img/bios/juhasz.jpg"><em>Dr. Levente Juhász</em><br>Levente, originally from Hungary, earned his PhD from the University of Florida. Currently he is with the GIS Center at the Florida International University in Miami, FL. Previously, he was a visiting scientist at the Joint Research Centre in Ispra, Italy, at Carinthia University of Applied Sciences in Austria and he also served as a data scientist for a geospatial startup, Mapillary. His research interests revolve around understanding the nature of user-generated geodata.
```
Comment (Nakaner) [inline program/index.md]: Could you please spell out "FL"? This makes it easier for non-Americans.

**Comment by MarcoMinghini** [comment]
>  do we even want this on the website?

I like it as well!

> should it be on a different page? the program page will later probably contain more stuff like the program schedule, so the page could get crowded at some point. on the other hand we have the left hand side menu now, so this should be less of an issue…

I agree that it should stay on the same page of the academic call for papers, and that the program-committee should do the same. No one is obliged to put a photo, but you can't imagine how useful is a photo for people that come to the conference ans want to meet the organizers :)
In addition, may also be a tab (or sub-tab) pointing to the "Committees" involved in the conference organization may be useful? this happens for instance for FOSS4G events (e.g. https://2019.foss4g.org/ under _about_ -> _committees_)

> should we also add email addresses?

No, but we should clearly state that any communication about the academic track should be directed to academic-sotm@openstreetmap.org.

> should we also add current affiliations?

Yes!

> should we tweak the bio texts? e.g. sometimes the bio starts with , sometimes it starts more formal: Dr. , some of them could maybe be a bit shorter, etc.

Here I disagree with @Nakaner :) I would try to make the bios a bit more coherent (similar length and similar type of information included). The whole page will look more clear in my opinion!

**Comment by tyrasd** [comment]
I've added the affiliations now and put the academic-sotm@openstreetmap.org on top. Also, the texts now all start with the first name. Could need more tweaking, though. Especially one persons stands out a bit having a much longer bio than the rest. But I'm now sure what exactly to remove… :sweat_smile:  do you think 58921c4 is ok?

**Comment by rabidllama** [comment]
Do you want me to go through and update the grammar etc. a little? Nothing major, just some things that could be done to make it read better

**Comment by tyrasd** [comment]
Sure, thanks @rabidllama!

**Comment by ChristineKarch** [comment]
I like the idea of explaining who the program committees are!

We could also do that for the general PC. Maybe in a reduced way (only the general program-sotm email address, no photos, ... we will discuss that). And probably not until the announcement of the call for abstracts at the end of February.

**Comment by MarcoMinghini** [comment]
@tyrasd I suggest to put myself and Yair as the first people in the list (on the top) to recognize our leading role, as it was done last year! thank you :)

**Comment by tyrasd** [comment]
sorry, I accidentally prematurely merged this PR. :see_no_evil: I've un-merged it in the repository, but I can't re-open this thread anymore.

I'll just go ahead and create a new page/tab and put the info about the committees there. Maybe we should call it "team", then we can also put names and information of the local&remote team members who are not part of any particular committee (similar to how it was done in the booklet in the past editions of the sotm). Does that sound ok?

**Pull request merged**